README.md

```diff
@@ -36,11 +36,11 @@ Amplify UI is an open-source UI library with cloud-connected components that are
 
 ## Component Matrix
 
-| **Connected Components** | **React** | **React Native** | **Angular** | **Vue** |
-| :----------------------- | :-------: | :--------------: | :---------: | :-----: |
-| Authenticator            |    ✅     |        🚧        |     ✅      |   ✅    |
-| In-App Messaging         |    ✅     |        ✅        |             |         |
-| Geo (MapView)            |    ✅     |                  |             |         |
+| **Connected Components** | **React** | **React Native** | **Angular** | **Vue** | **Flutter** | **Android** | **Swift** |
+| :----------------------- | :-------: | :--------------: | :---------: | :-----: | :---------: | :---------: | :-----: |
```

Comment:
Should we be adding these to our main repo since the packages are not web related?

Reply:
Good point